package.json

```diff
@@ -25,6 +25,7 @@
     "start": "cross-env NODE_ENV=production node ./build/server.js",
     "start:e2e": "cross-env NODE_ENV=test node --require dotenv/config ./build/server.js",
     "test": "cross-env SECRETS_OVERRIDE=1 DATABASE_URL='mysql://root:root_password@127.0.0.1:3306/starchart_test' NODE_OPTIONS=--require=dotenv/config vitest",
+    "e2e": "cross-env PORT=8080 DATABASE_URL='mysql://root:root_password@127.0.0.1:3306/starchart_test' start-server-and-test dev http://localhost:8080 \"playwright test --ui\"",
```

Comment:
Should we name this something else, similar to `test:e2e:dev`? Or would the name get too long?

Reply:
I hate these names, I can never remember what they are.  I want something that's easy to use and means "I'm a dev who wants to run the e2e tests..."  The long names are fine for scripts/CI.  What do you think?

Reply:
Yea, that makes sense. The names are quite long